src/styl/layouts/support-form.styl

```diff
@@ -4,27 +4,122 @@
   display: block
   .support-form__inputs
     opacity: 0
-    transition: opacity var(--transition-speed-medium)
   &--active
     .support-form__inputs
       opacity: 1
 
-.support-form__button, .support-form__custom-input, .support-form__submit
-  color: var(--color-red)
+.support-form__schedule-option-right
+  border-radius:  0px 4px 4px 0px;
```

Comment:
in v0.11.0 of stylus library there is a css variable for this. probably worth updating the dependency in this feature and using that var here